examples/gno.land/p/demo/blog/blog.gno

```diff
@@ -42,11 +42,20 @@ func (b Blog) RenderHome(res *mux.ResponseWriter, req *mux.Request) {
 	}
 
 	res.Write("<div class='columns-3'>")
+
+	var sorted []*Post
```

Comment:
I would personally make it so we modify the contract to store the blog posts in order. (Say, `[]*Post` inside of the BlogType?). When a post is added/deleted, update that accordingly.

The reason I'd make this is that, I think the final destination for the blog is the main website (gno.land) which will run the Portal Loop. `CreatedAt` works by getting `time.Now()` when publishing.

On the portal loop, it is my understanding that at each deployment (= new commit on the repo), the website is upgraded to the latest master branch, and all the transactions from the "previous" portal loop are re-played at genesis. Thus; `CreatedAt` would always be set to the same date.

I think as a consequence of this, we may also need to change `CreatedAt` to be configurable and something that is not generated by `time.Now()`, but rather by the caller, so that the posts show up with the correct information.

So by making sure we are adding a `*Post` to a slice each time, we can be sure that even if `time.Now()` is unreliable, the posts will show up in the correct order.

cc/ @zivkovicmilos who may be able to correct me if I said something wrong.

Reply:
The `CreatedAt` (or published date? I don't remember) field is set manually by the `gnoblog-cli` client, so we can trust it in this case.

However, your analysis is logical and we should attempt to address it for other contracts, particularly the `boards` one. We should consider incorporating the block timestamp into the export/import feature of the portal loop during our iterations.

Reply:
CreatedAt/UpdatedAt don't seem to be changed by Gnoblog: https://github.com/gnolang/gno/blob/d421b963aed7f7c3ba3718edfc6fbd787fa8f0dd/examples/gno.land/p/demo/blog/blog.gno#L139